modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/components/Carrousel/Galerry.tsx

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import React, {useState} from 'react';
+
+import './index.scss';
+
+interface DamageSummaryProps {
+	images: string[];
+	size: {
+		height: string;
+		width: string;
+	};
+}
+
+const Galerry: React.FC<DamageSummaryProps> = ({images, size}) => {
+	const [modalOpen, setModalOpen] = useState(false);
+	const [activeImage, setActiveImage] = useState(0);
+
+	const hasNextImage = activeImage < images?.length - 1;
+	const hasPreviousImage = activeImage > 0;
+
+	const handleImageClick = (index: number) => {
+		setActiveImage(index);
+		setModalOpen(true);
+	};
+
+	const handleModalClose = () => {
+		setModalOpen(false);
+	};
+
+	const handlePrevClick = () => {
+		if (hasPreviousImage) {
+			setActiveImage(activeImage - 1);
+		}
+	};
+
+	const handleNextClick = () => {
+		if (hasNextImage) {
+			setActiveImage(activeImage + 1);
+		}
+	};
```

Comment:
```suggestion
	  
	  const handleClick = () => {
		    if (hasPreviousImage) {
			setActiveImage(activeImage - 1);
		}
		    if (hasNextImage) {
			  setActiveImage(activeImage + 1);
		  }
	  };
```

Reply:
Nicee! Thx!!!

Reply:
I believe that is not the better approach for this case.